### DISCUSSION TITLE
When will the 0.5.0 build released? Will there be prereleases?

### DISCUSSION THREAD

**Discussion by manuelwallrapp** [tektoncd/results#375] [Q&A] [answered]
Hi

When will you release the 0.5.0 and will you create pre releases? Meanwhile I try to build from master branch, since version 0.4.0 is quite behind with the features. I also would like to test the GRPC without RBAC Authentication since I have my difficulties to authenticate myself. For the moment I built it locally and installed it.

Best regards

Manu

**Comment by avinal** [comment, marked as answer]
Here is the Release Tracker for 0.5.0: https://github.com/tektoncd/results/issues/356

**Reply by avinal** [reply]
0.5.0 is up https://github.com/tektoncd/results/releases/tag/v0.5.0 💐
